components/FoodCard/FoodCard.test.js

```diff
@@ -12,12 +12,12 @@ test("render the time and comment", () => {
   render(<FoodCard food={testFood} />);
 
   const name = screen.getByText(/Winterlicher Blattsalat/i);
```

Comment:
If you change it to an `h2` (or similar), you can use the `getByRole` query here.

Reply:
Thanks. I changed it to "NameOfPage"